src/pyhf/constraints.py

```diff
@@ -116,7 +111,19 @@ def logpdf(self, auxdata, pars):
 
         result = prob.Independent(
             prob.Normal(normal_means, self.sigmas), batch_size=self.batch_size
-        ).log_prob(self._dataprojection(auxdata))
+        )
+        return result
+
+    def logpdf(self, auxdata, pars):
```

Comment:
Methods should have docstrings.

Reply:
done

Reply:
The args don't match the signature though.

Reply:
Fixed this.